### PR TITLE
Some UI features

### PR DESCRIPTION
**Pull request by maliksenpai**
- Added hover effect for icon buttons 
- Added shadow for sidebar
 - Added listen clickoutside for sidebar

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -105,8 +105,8 @@ body {
 .container {
     width: min(99%, 90.5rem);
     margin-inline: auto;
-    overflow-x: scroll;
-    min-height: 85%;
+    overflow-x: auto;
+    min-height: 95vh;
```
Comment (waterrmalann) [inline css/main.css:109]: This seems to have a side effect of pushing the horizontal scrollbar too low that it's not visible at all.

For accessibility reasons, it's desirable to keep the scrollbar visible.

File: css/main.css [hunk 2]
```diff
@@ -165,9 +165,23 @@ body {
     letter-spacing: 0.7rem;
 }
 
+.fa {
```
Comment (waterrmalann) [inline css/main.css:168]: Really love the hover effect.

It did seem to increase the size (height) of each individual cell of a card. I personally preferred the more compact cards like before but I like this too, perhaps an updated theme (which I plan to work on later) would make it look better.

File: js/main.js [hunk 3]
```diff
@@ -793,12 +793,22 @@ e_deleteButton.addEventListener('click', () => {
 
 /* <=================================== Sidebar ===================================> */
 function toggleSidebar() {
+    console.log("malik", 123);
```
Comment (waterrmalann) [inline js/main.js]: I feel like you could get rid of this console log line here.

**Review by waterrmalann — COMMENTED** [review]
Really appreciate the pull request!

I'd love to merge some of these changes to the main branch. I've left some comments on some things, you could check it out and maybe fix those.

**Comment by maliksenpai** [comment]
Hello @waterrmalann ,

Can you check again my pr?

File: css/main.css [hunk 4]
```diff
@@ -106,7 +106,7 @@ body {
     width: min(99%, 90.5rem);
     margin-inline: auto;
     overflow-x: auto;
-    min-height: 95vh;
+    min-height: 93vh;
```
Comment (waterrmalann) [inline css/main.css:109]: This still does not fix the issue of the invisible horizontal scrollbar on my device. I feel like the old percentage value of 85% would be most compatible for min-height instead.

**Review by waterrmalann — CHANGES_REQUESTED** [review]
One more change and I think it's good to merge :)

**Comment by maliksenpai** [comment]
@waterrmalann I reverted min-height. Also is there anything to do after that?

**Review by waterrmalann — APPROVED** [review]

**Comment by waterrmalann** [comment]
> @waterrmalann I reverted min-height. 

Nice work! All good now :)

> Also is there anything to do after that pr?

Not much else here but if you wish to help, you could check out the open [issues](https://github.com/waterrmalann/kards/issues).

**Pull request merged**